adapters/mqtt-base/src/test/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapterTest.java

```diff
@@ -472,6 +472,41 @@ public void testHandlePublishedMessageFailsForMalformedTopic() {
         verify(endpoint, never()).publishAcknowledge(anyInt());
     }
 
+    /**
+     * Verifies that the adapter discards messages whose connection is expired.
+     */
+    @Test
+    public void testHandlePublishedMessageFailsForExpiredConnection() {
```

Comment:
...FailsForExpiredToken ?

The MQTT connection can also expire if the heart beat is not sent but that is a different thing FMPOV.

Reply:
Good point, renamed the test.